draft-liu-acme-rats.md

```diff
@@ -141,36 +157,43 @@ An example extended Authorization Object (that contains a Challenge Object):
 ~~~~~~~~~~
 
 
-# Extensions -- rats challenge type
+# Extensions -- rats challenge types
 
 A rats challenge type help the Client prove ownership to its attestation result identifier. This section describes the challenge/response extensions and procedures to use them.
 
-## RATS-01 Challenge {#rats01}
+## device-attest-02 Challenge {#rats01}
 
-RATS-01 Challenge simply works with Passport Model of RATS. The corresponding Challenge Object is:
+device-attest-02 Challenge simply works with Passport Model of RATS. The corresponding Challenge Object is:
 
 type (required, string):
-: The string "rats-01".
+: The string "device-attest-02".
 
 url (required, string):
 : The URL that the Client post its response to.
 
 token (required, string):
 : Same as Section 8.3 of RFC8555.
 
+nonce (optional, string):
+: If attestation freshness is required, then the Server MAY present a nonce which then MUST be echoed in the provided attestation. In some situations, the nonce will come from a separate RATS Verifier, and therefore needs to be a distinct value from the ACME token.
+
+attestClaimsHint (optional, list of string)
+: If the Server requires attestation of specific claims or properties in order to issue the requested certificate profile, then it MAY list one or more types of claims from the newly-defined ACME Attest Claims Hints registry defined in {{sec-claimshints}}.
+
 The response sent to the url is:
 
 ~~~~~~~~~~
-keyAuthorization = token || '.' || base64url(attestationResult)
+keyAuthorization = token || '.' || cmw
 ~~~~~~~~~~
 
+where `cmw` MAY be either a CMW in JWT format, or a Base64 CMW in CWT format as per {{CMW}}.
+
```

Comment:
mike, in the sessions you mentioned to return the cmw in a proper JWS formatted challenge-response, which I think may be a better place to encapsulate it. I revised this in PR #6 . what do you think? Other revisions in his PR looks good. we can cherry-pick them and merge 2 PRs